packages/salesforcedx-vscode-core/src/orgBrowser/metadataType.ts

```diff
@@ -21,6 +21,13 @@ type MetadataObject = {
   xmlName: string;
 };
 export class TypeUtils {
+  public static readonly FOLDER_TYPES = new Set([
```

Comment:
Are these the only types that support folders ?

Reply:
Sources:
https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_folder.htm
https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_listmetadataquery.htm